GiEnJul/ClientApp/src/components/admin/common/RecipientTable.tsx

```diff
@@ -72,7 +71,20 @@ const DatatableRecipient: React.FC<Props> = ({ data }) => {
   return (
     <Container>
       {data.map((recipient, index) => (
-        <Accordion key={`acc_recipient_${index}`}>
+        <Accordion
+          onChange={() =>
+            handleRecipientChange(recipient.rowKey, recipient.partitionKey)
+          }
+          key={`acc_recipient_${index}`}
+          style={
```

Comment:
Not sure if this should be inline CSS or not. Could probably be added to a hook and used as a class instead?

Reply:
This is work in progress. Will be separated in a new file